lib/everett/subject.rb

```diff
@@ -33,7 +33,9 @@ def add_observer(observer)
       callbacks = CALLBACKS.select { |callback| observer.respond_to?(callback) }
       return false if callbacks.empty?
 
-      callbacks.each { |callback| @model.public_send(callback, observer) }
+      # TODO: Rewrite the following line in the form of passing an observer instance to callbacks
+      # once ActiveRecord makes after_{create,update,destroy}_commit accept callback objects.
+      callbacks.each { |callback| @model.public_send(callback) { observer.public_send(callback, self) } }
```

Comment:
__[rubocop]__
* Line is too long. [105/100] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->